src/pages/Dashboard.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { useEffect, useState } from 'react';
 import NoOrganization from '../components/NoOrganization';
-import userService, { UserOrganization } from '../services/user.service';
```

Comment:
It seems that we have an Organization type in `user.service` and in `organization.service` can we reduce this to just having the type in `organization.service`?

Reply:
I think having organization.service import User from user.service and user.service importing Organization causes 'Dependency cycle' not sure if there's a way around that.

Reply:
Actually at that point I think it makes sense to move those organization functions out of user service and into organization service.

Reply:
@kpjVideo I agree with that.